lib/web-editor-uri-mini-view.coffee

```diff
@@ -1,70 +1,11 @@
-validator = require 'validator'
 Point = require 'atom'
-{$, View, TextEditorView} = require 'atom-space-pen-views'
```

Comment:
Removed the deprecated protocol/application programming interface entirely. Nothing is require(d) except 'atom'